media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```
@@ -0,0 +1,128 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+_COMMIT="a97b03fad"
+MY_PV="${PV}-${_COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server-new"
+#URI="https://artifacts.plex.tv/plex-media-server-experimental/"
+SRC_URI="
+	amd64? ( ${URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="system-openssl"
+
+BDEPEND="dev-util/patchelf"
+
+RDEPEND="
+	acct-group/plex
+	acct-user/plex
+	system-openssl? ( dev-libs/openssl )
+	"
+# rdepend not working yet
+#custom-ffmpeg? ( media-video/plex-ffmpeg )
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=( "usr/lib/plexmediaserver/lib/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so" )
+
+PATCHES="${FILESDIR}/plexmediaserver.desktop.patch"
```

PATCHES is an array by convention, even though Bash muddies the waters.
```suggestion
PATCHES=(
    "${FILESDIR}/plexmediaserver.desktop.patch"
)
```

We should also version the patch and prefix it with the package name. I usually version it with the first version I based it on / the version it applied to first. We can reuse the same patch in later ebuilds without renaming it.
```suggestion
PATCHES=(
    "${FILESDIR}/${PN}-1.22.2.4282-desktop-file.patch"
)
```